tools/export/iar/__init__.py

```diff
@@ -120,22 +121,13 @@ def generate(self):
         self.gen_file('iar/ewd.tmpl', ctx, self.project_name + ".ewd")
         self.gen_file(self.get_ewp_template(), ctx, self.project_name + ".ewp")
 
-    def build(self):
+    @staticmethod
+    def build(project_name, clean=True):
```

Comment:
The rest of our APIs use the parameter `clean` to mean  "delete a bunch of stuff before doing the thing I asked you to do". But in this case, it means "delete all the stuff you created when you're done doing the thing I asked you to do".

For this reason, maybe we should call this parameter `cleanup` instead of `clean`?